### PR TITLE
Add bounding boxes to the specialized form parser and adjust test data

### PR DESCRIPTION
**Pull request by asrivas**

### CODE REVIEW

File: specialized/specialized_form_parser.ipynb [hunk 1]
```diff
@@ -154,6 +152,44 @@
     "doc = process_document_sample()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Draw the bounding boxes¶\n",
+    "We will now download the pdf above a jpg and use the spatial data to mark our values."
```
Comment (camushoilingma) [inline specialized/specialized_form_parser.ipynb:160]: "as a jpg"

or maybe more accurate, something like:
"We will now use the spatial data returned by the processor to mark our values on the invoice pdf file that we first converted into a jpg."

File: specialized/specialized_form_parser.ipynb [hunk 2]
```diff
@@ -101,12 +101,8 @@
     "    document = result.document\n",
     "    entities = document.entities\n",
     "    print(\"Document processing complete.\\n\\n\")\n",
-    "    \n",
```
Comment (camushoilingma) [inline specialized/specialized_form_parser.ipynb:104]: Why did we remove this but keep the rest of the HITL code in the bottom?

**Review by camushoilingma — CHANGES_REQUESTED** [review]

File: specialized/specialized_form_parser.ipynb (hunk 1 above)
Comment (asrivas) [reply specialized/specialized_form_parser.ipynb]: Done

File: specialized/specialized_form_parser.ipynb (hunk 2 above)
Comment (asrivas) [reply specialized/specialized_form_parser.ipynb:104]: How very strange, if you look at the full diff of my most recent commit it's in there

**Review by camushoilingma — APPROVED** [review]

**Pull request merged**